web/locales/de.json

```diff
@@ -27,6 +27,7 @@
         "back": "zurück",
         "source": "Quelle",
         "fossilfuels": "fossile Brennstoffe",
+        "carbonintensity": "CO2 Konzentration",
```

Comment:
I'm all for merging fast so let's not wait for anything else, but on the other hand let's not spend time making multiple pull requests if other changes do not make this PR unreadable.